### DISCUSSION TITLE
kanban [May 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2144] [Discord Threads]
> 🙋‍♂️ **Juanjomarg**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### it was at proof of concept during launch, its not yet available at the moment  
#  
> 🙋‍♂️ **Juanjomarg**  
#### Thanks for the quick reply <@310157751395287040> , cheers  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/975816502785753128)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
